docs/serverless/08-05-create-git-function.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Create a Function from Git repository sources
+type: Tutorials
+---
+
+This tutorial shows how you can create a Function in an alternative way -  building it from code and dependencies stored in a Git repository instead of keeping the code in the Function CR. The tutorial is based on the Function from the [`orders service` example](https://github.com/kyma-project/examples/tree/master/orders-service). It describes steps required to fetch Function's source code and dependencies from a public Git repository that does not require any authentication method. However, it also provides additional guidance on how to secure it if you are using a private repository.
```

Comment:
You can rephrase it like this to avoid using the dash (just a preferential suggestion):
```This tutorial shows how you can build a Function from code and dependencies stored in a Git repository, which is an alternative way to keeping the code in the Function CR.```